netconf/transport_ssh.go

```diff
@@ -149,12 +148,12 @@ func DialSSHTimeout(target string, config *ssh.ClientConfig, timeout time.Durati
 		}
 	}()
 
-	return NewSession(t), nil
+	return t, nil
 }
 
-// NoDialSSH - create a new NETCONF session over the given ssh Client.
-func NoDialSSH(sshClient *ssh.Client, options ...TransportSSHOption) (*Session, error) {
-	t := new(TransportSSH)
+// NoDialSSH - create a new TransportSSH from given ssh Client.
+func NoDialSSH(sshClient *ssh.Client) (*TransportSSH, error) {
```

Comment:
The logging PR is none breaking. If we merge this PR it will break, are you OK with that?

Maybe we should tag the current main with logging before we break it?

Reply:
I will create a new release and capture this in the release note